libr/core/project.c

```diff
@@ -164,7 +164,8 @@ R_API int r_core_project_list(RCore *core, int mode) {
 }
 
 R_API int r_core_project_delete(RCore *core, const char *prjfile) {
-	char *path;
+	char *path, *tmp, *prjDir;
```

Comment:
camelCase is not used a lot in the codebase

Reply:
I used it because i saw it in other functions in the same file. Should I change it to something like `prj_dir`?

Reply:
ok, you're right then sorry